x/ibc/23-commitment/types/merkle.go

```diff
@@ -379,8 +379,12 @@ func (proof MerkleProof) ValidateBasic() error {
 
 // validateVerificationArgs verifies the proof arguments are valid
 func (proof MerkleProof) validateVerificationArgs(specs []*ics23.ProofSpec, root exported.Root) error {
-	if proof.Empty() || root == nil || root.Empty() {
-		return sdkerrors.Wrap(ErrInvalidMerkleProof, "empty params or proof")
+	if proof.Empty() {
+		return sdkerrors.Wrap(ErrInvalidMerkleProof, "proof cannot be empty")
+	}
+
+	if root == nil || root.Empty() {
```

Comment:
ran into this once, I thought it was useful to directly indicate which part is wrong instead of having to debug it